pos_default_customer/__openerp__.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Point of Sale Default Customer',
+    'version': '8.0.0.1',
+    'category': 'Point Of Sale',
+    'summary': 'Point of Sale Default Customer',
+    'author': 'Digital5 S.L.',
```

Comment:
Please review template and OCA
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L10